src/gui/CMakeLists.txt

```diff
@@ -286,6 +295,7 @@ mdserverproxy
 avtdbatts
 winutil
 ${GUI_QT_LIBS}
+$<$<BOOL:${VISIT_ANARI}>:anari::anari>
```

Comment:
```suggestion
$<$<BOOL:${HAVE_ANARI}>:anari::anari>
```

Reply:
Added `set(HAVE_LIBANARI true)` like OSPRay did. Just having `add_definitions(-DHAVE_ANARI)` doesn't work with the CMake expression.